common/internal_model/src/main/scala/weco/catalogue/internal_model/work/Work.scala

```diff
@@ -222,20 +216,6 @@ object WorkState {
     val relations = Relations.none
 
     override val modifiedTime: Instant = sourceModifiedTime
-
-    def internalWorksWith(version: Int): List[Work.Visible[Identified]] =
```

Comment:
I moved this into the merger because that's the only place it's used, and the next time we tweak it I don't want to cause a full rebuild of internal model.